packages/react-composites/tests/app/lib/MockCallAdapter.ts

```diff
@@ -152,7 +152,24 @@ export class MockCallAdapter implements CallAdapter {
   }
 
   blurVideoBackground(): Promise<void> {
-    throw new Error('blurVideoBackground not implemented.');
+    this.modifyState((draft: CallAdapterState) => {
+      if (!draft.call) {
+        throw new Error('Call not found');
+      } else if (draft.call.localVideoStreams.length > 0) {
+        draft.call.localVideoStreams = [
```

Comment:
here `blurBackground` is setting the video stream to something new. Should it not only be updating fields related to video effects?

Reply:
The blurVideoBackground adapter method will call the calling sdk method which can't be done for the MockCallAdapter. I guess it will be hard to mock without directly setting the video stream in the call adapter state.

Reply:
Rather than replacing the video stream, I am replacing the view now.